notebooks/MAST/TESS/beginner_how_to_use_tp/beginner_how_to_use_tp.ipynb

```diff
@@ -0,0 +1,354 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Break down a pixel value of 267 (yellow pixels displayed above) into its\n",
+    "# constituent bits.\n",
+    "bits = np.binary_repr(267)\n",
+    "print(bits)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Binary bits start from the right and end at the left, so the bit farthest on the right is the Least Significant Bit (LSB, equal to 2^0 = 1), the second from the right is 2^1=2, the third from the right is 2^2=4, etc. This bit mask has bit values of 1 (2^0), 2 (2^1), 8 (2^3), and 256 (2^8) set. From our look-up table, these values mean the pixels have been: collected by the spacecraft (value = 1), used in the optimal photometric aperture (value = 2), used to calculate the flux-weighted centroid (value = 8), and are on CCD Output D (value = 256)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Marking All Pixels Used In The Photometric Aperture\n",
+    "\n",
+    "Let's display the pixels in the aperture FITS extension again, but this time, let's only mark those that are used in the optimal photometric aperture (bit mask value of 2 set).  While it is possible to do this in fewer lines, we'll show more of the steps involved for clarity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create an array that will keep track of which pixels have a bitmask\n",
+    "# value of 2 set. To start, everything is set to 0 = not set.  We make it\n",
+    "# the same dimension as the aperture pixel table, for clarity.\n",
+    "bitmask2_set = np.zeros(aperture.shape)\n",
+    "# Identify which pixels have the bit mask value of 2 set.  We'll first\n",
+    "# loop over each row (i) in the 11x11 table, and then each column (j).\n",
+    "for i,row in enumerate(aperture):\n",
+    "    for j, pix in enumerate(row):\n",
+    "        # Get the bitmask as a string of zeros and ones.\n",
+    "        this_bitmask = np.binary_repr(pix)\n",
+    "        # Is the bitmask value of 2 set?  If so, the 2nd integer from the\n",
+    "        # right should be set to 1.  We'll use Python negative indexes to\n",
+    "        # access the second integer from the right.\n",
+    "        if this_bitmask[-2] == '1':\n",
+    "            # Then record this pixel as having a bitmask value of 2 set.\n",
+    "            bitmask2_set[i,j] = 1"
+   ]
```

Comment:
Suggest adding newlines before each comment line (or group of comment lines) to improve readability of the cell.